### ISSUE TITLE
arXiv-2019/03-Fine-tune BERT for Extractive Summarization

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#128]
**一句话总结：**

**资源：**

- [pdf](https://arxiv.org/abs/1903.10318)
- [code](https://github.com/nlpyang/BertSum)
- [paper-with-code](https://paperswithcode.com/paper/fine-tune-bert-for-extractive-summarization-1)

**关键字：**

- dataset: CNN/Dailymail and NYT datasets
- 

**笔记：**

The task is often divided into two paradigms, abstractive summarization and extractive summarization. In abstractive summarization, target summaries contains words or phrases that were not in the original text and usually require various text rewriting operations to generate, while extractive approaches form  summaries by copying and concatenating the most important spans (usually sentences) in a document. In this paper, we focus on extractive summarization.

- abstractive summarization
   - 生成的摘要可能不包含原文中的单词或短语，需要rewrite.
- extractive summarization
   - 从文本中copy并concat重要的span。

![image](https://user-images.githubusercontent.com/10768193/56584798-7ed9e180-6617-11e9-8110-b15ade281660.png)

lr的更新是通过warming-up实现的

![image](https://user-images.githubusercontent.com/10768193/56584859-9fa23700-6617-11e9-8f01-263cdcd5f1b8.png)

使用了 gradient accumulation per two steps，batch_size为36. 

**模型图：**

![image](https://user-images.githubusercontent.com/10768193/56584672-33273800-6617-11e9-8041-826c89e50fc5.png)

**结果**：

![image](https://user-images.githubusercontent.com/10768193/56585551-04aa5c80-6619-11e9-8b76-1dacc023c415.png)

**接下来要看的论文**：
